Src/IronPython.Modules/re.cs

```diff
@@ -969,8 +962,26 @@ internal class ParsedRegex {
                                                 pattern = pattern.Substring(0, tmpIndex) + ">" + pattern.Substring(tmpIndex + 1);
                                             } else {
                                                 containsNamedGroup = true;
+                                                
+                                                // we need to look and see if the named group was already seen and throw an error if it was
+                                                if(nameIndex + 1 < pattern.Length && pattern[nameIndex + 1] == '<') {
+                                                    int tmpIndex = nameIndex + 2;
```

Comment:
Isn't this just `string.IndexOf`?

Reply:
Yeah, just got into the mode of reusing what was above for some reason.